README.md

```diff
@@ -30,6 +25,7 @@ Options:
   -v, --version            output the version number
   -i, --icon <icons...>    A folder path which stores SVG icons. Multiple folders can be set.
   -r, --ratio <ratios...>  pixel ratio to generate sprite. default is 1.
+  --sdf                    generate sprite with signed distance field. (default: false)
```

Comment:
```suggestion
  --sdf                    generate sprite with SDF (Signed Distance Field). (default: false)
```

I think it would  also be nice to have abbreviation of SDF

Reply:
Also I resolve it.